common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal_types.h

```diff
@@ -1,6 +1,6 @@
 /* SPDX-License-Identifier: BSD-2-Clause-Patent
  *
- * Copyright (c) 2016-2019 Intel Corporation
+ * Copyright (c) 2019 Intel Corporation
```

Comment:
This should be done in the commit which added this file (its a new one). Also, its 2020 now :)

Reply:
🎉

Reply:
fixed